BackBLAST

```diff
@@ -105,7 +103,8 @@ function perform_setup() {
     printf "   -x genome_extension: extension for predicted protein files of subject genomes [default: faa]\n"
     printf "   -@ threads: maximum threads to use for any process [default: 1]\n\n"
     printf "Advanced parameters (use with care):\n"
-    printf "   -T template_config: Path to the template config file used in setup [default: ${TEMPLATE_CONFIG}]\n\n"
+    printf "   -T template_config: Path to the template config file used in setup [default: ${TEMPLATE_CONFIG}]\n"
+    printf "   -U utils_dir: Path to the BackBLAST utility script directory [default: ${UTILS_DIR}]\n\n"
```

Comment:
User can change the location of the utility script directory if they want to (advanced setting; most people will never need this).